gcovr/__main__.py

```diff
@@ -29,1024 +29,23 @@
 # $Date$
 #
 
-try:
-    import html
-except ImportError:
-    import cgi as html
 import copy
 import os
 import re
-import subprocess
 import sys
-import time
-import xml.dom.minidom
-import datetime
-import zlib
 
 from optparse import Option, OptionParser, OptionValueError
-from string import Template
 from os.path import normpath
 
-try:
-    xrange
-except NameError:
-    xrange = range
+from .gcov import get_datafiles, process_existing_gcov_file, process_datafile
+from .utils import get_global_stats, build_filter
+from .version import __version__, version_str  # noqa: F401 unused __version__
```

Comment:
Why import `__version__` that requires `# noqa: F401 unused __version__` ?

Reply:
🤔 hmm, good question. I just cargo-culted the existing code here because I'm not 100% sure about the Python conventions in this case. I see two alternatives:

- import the version in the `__init__` so that we have a `gcovr.__version__` attribute.
- leave the version in the `gcovr.version` module until we offer an API that would profit from a version attribute.

What do you think would be best?

Reply:
On second thought, I'll keep `__version__` but remove the `version_str()` function which no longer adds any value over accessing the `__version__` directly.